lib/money/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 class Money
-  VERSION = "3.2.1"
+  VERSION = "3.2.2"
```

Comment:
Gotta bump the shame version

![SemVer shame version](https://github.com/user-attachments/assets/14660825-6235-4623-80f5-af7c546e7e6c)